db/migrations/core-data/20230317152843.do.hdf-confirm-codes.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO mtc_admin.hdfStatus ([description], hdfStatusCode) VALUES
```

Comment:
This is a lookup table, I would prefer the table to end with `Lookup` to make that clear.

Reply:
then we have an issue with the `jobStatus` table also